envs/monkey_zoo/packer/setup_mimikatz_14.yml

```diff
@@ -36,39 +35,34 @@
         state: present
         enabled: yes
 
-    - name: Disable Windows Defender using registry
-      win_regedit:
-        path: HKLM:\SOFTWARE\Policies\Microsoft\Windows Defender
-        name: DisableAntiSpyware
-        data: 1
-        type: dword
-        state: present
-
-    - name: Disable Windows Defender
-      win_shell: Set-MpPreference -DisableIntrusionPreventionSystem $true -DisableIOAVProtection $true -DisableRealtimeMonitoring $true -EnableNetworkProtection AuditMode -Force
-      become: yes
-      become_user: m0nk3y
-
     - name: Create a generic credential for mimikatz-15
-      win_credential:
-        name: 10.2.2.15
-        type: generic_password
-        username: m0nk3y
-        secret: pAJfG56JX><
-        state: present
+      win_command: "cmdkey /add:10.2.2.15 /user:{{ user_name }} /pass:\"pAJfG56JX><\""
       become: yes
       become_user: m0nk3y
 
-    - name: Change the hostname to mimikatz-14
+    # Disallow RDP to this machine from internal ips
+    - import_tasks: tasks/windows_disallow_rdp_for_subnet.yml
+      vars:
+        subnet_ip: "10.2.0.0/16"
+
+    - import_tasks: tasks/windows_enable_automatic_logon.yml
+      vars:
+        username: "{{ user_name }}"
+        password: "{{ user_password }}"
+
+    - name: Change the hostname to {{ host_name }}
       ansible.windows.win_hostname:
-        name: mimikatz-14
+        name: "{{ host_name }}"
       register: res
 
     - name: Reboot
       ansible.windows.win_reboot:
       when: res.reboot_required
 
-    - import_tasks: tasks/windows_disable_powershell_remoting.yml
+    # Disallow PowerShell to this machine from internal ips
+    - import_tasks: tasks/windows_disallow_powershell_remoting_on_subnet.yml
+      vars:
+        subnet_ip: "10.2.0.0/16"
```

Comment:
It would be best if all firewall rules followed each other (move this after `tasks/windows_disallow_rdp_for_subnet.yml`)